Documentation/ExtensionArchitecture/ComposerJson/Index.rst

```diff
@@ -187,9 +200,23 @@ require
 (*required*)
 
 At the least, you will want to require `typo3/cms-core`.
-You can add other system extensions and third party extensions,
+You should add other system extensions and third party extensions,
```

Comment:
```suggestion
You must add other system extensions and third party extensions,
```

You'll get errors if you don't do this.

Reply:
You might not depend on other system extensions.